app/forms/idv/document_capture_form.rb

```diff
@@ -28,11 +28,16 @@ def self.model_name
     def submit(params)
       consume_params(params)
 
-      FormResponse.new(success: valid?, errors: errors.messages)
+      FormResponse.new(success: valid?, errors: errors.messages, extra: extra_hash)
     end
 
     private
 
+    def extra_hash
```

Comment:
in other files we call this `extra_analytics_attributes` -- can we do the same here for consistency?